src/scripts/plugin/videojs.vast.vpaid.js

```diff
@@ -362,23 +366,46 @@ module.exports = function VASTPlugin(options) {
       }
 
       player.controlBar.addChild('AdsLabel');
+      if (vastResponse.icons && vastResponse.icons.length > 0) {
+        adIconIntegrator = new IconIntegrator(player);
+        adIconIntegrator.renderIcons(vastResponse.icons, callback);
+      }
+
+      var elems = document.getElementsByClassName('vjs-next-button');
+      if (elems && elems.length > 0) {
+        playlistNextButton = elems[0];
+        playlistNextButton.style.display = 'none';
+      }
     }
 
     function removeAdsLabel() {
     	if (window._molSettings.playsInBreak) {
     		player.off('timeupdate', updateTimeControls);
     		if (window._molSettings.timeOffset) {
-          player.controlBar.getChild('currentTimeDisplay').el_.children[0].style.display = 'block';
+          showOriginalTimeDisplay(true);
           var bcTimeDisplay = document.getElementById('bc_time_display');
           if (bcTimeDisplay) {
             bcTimeDisplay.style.display = 'none';
           }
     		}
     	}
       player.controlBar.removeChild('AdsLabel');
+      if (playlistNextButton) {
+        playlistNextButton.style.display = '';
```

Comment:
Why are you setting display to `''`? Shouldn't it be one of `'none'`, `'inline'`, `'block'` or `'inline-block'`?

Reply:
Setting display to '' resets the element's display property to the default.